src/packages/@ncigdc/containers/ConsequencesTable.js

```diff
@@ -77,7 +77,12 @@ const ConsequencesTableComponent = compose(
           consequence: transcript.consequence_type,
           coding_dna_change: transcript.annotation.hgvsc,
           strand: transcript.gene.gene_strand
-            ? strandIconMap[transcript.gene.gene_strand.toString(10)]
+            ? <span>
+                {strandIconMap[transcript.gene.gene_strand.toString(10)]}
+                <ForTsvExport>
```

Comment:
I think it will potentially cause confusion to have some TSV export buttons export all data, and other TSV export buttons only export the current page, with no way for the user to distinguish how the same button will behave, and no good way to communicate why the limitation exists for some tables and not others (users shouldn't have to know about client-side vs server-side pagination). 

Imo it's better to have predictable behavior, and defer this feature until we have the backend solution for TSV exports that will support exporting all rows for all tables